tools/standardize_config_json.py

```diff
@@ -104,24 +103,57 @@ def print_diff(original: collections.OrderedDict, new: collections.OrderedDict):
     LOG.info("\n".join(diff))
 
 
-def process(filename: str):
-    contents = read_json_file(filename)
-    original = read_json_file(filename)
+def process(config_json_path: str, configuration: dict):
+    contents = read_json_file(config_json_path)
+    original = read_json_file(config_json_path)
 
     reordered = reorder_service(contents["properties"]["service"]["properties"])
     contents["properties"]["service"]["properties"] = reordered
 
+    sections = configuration.get("sections", {})
+    for section_name, head_and_tail in sections.items():
+        head = head_and_tail["head"]
+        tail = head_and_tail["tail"]
+        properties = contents["properties"][section_name]["properties"]
+        reordered = reorder(properties, head, tail, reorder_property)
+        contents["properties"][section_name]["properties"] = reordered
+
     print_diff(original, contents)
 
-    write_json_file(filename, contents)
+    write_json_file(config_json_path, contents)
 
 
 if __name__ == "__main__":
-    files = [
-        "frameworks/cassandra/universe/config.json",
-        "frameworks/hdfs/universe/config.json",
-        "frameworks/helloworld/universe/config.json",
-    ]
+    args = sys.argv[1:]
```

Comment:
Do we not want to check the length of args before doing this? Also, since argparse is part of the standard library these days we could use that instead to avoid checks on arguments and specific handling of the help arguments and specific usage docs.

Reply:
Fixed in 072eb00997b0f6d39c184bcd8501dc19f34db4e9.